boot/zephyr/Kconfig

```diff
@@ -68,7 +68,6 @@ config SINGLE_APPLICATION_SLOT
 
 choice
 	prompt "Signature type"
-	default BOOT_SIGNATURE_TYPE_ECDSA_P256 if HAS_HW_NRF_CC310
```

Comment:
I don't think you should disable this default config, since I would assume most users use Zephyr with west (or should!). As long as the message states clearly how to fix the problem, it should be fine for everyone.

Reply:
Maybe we could mention in the readme that `you can also use west` (in case user is not aware about this)?

Reply:
I guess I was too fast at commenting before, but it doesn't seem that Zephyr's west brings in `nrfxlib`, so you can disregard my previous comment.